test/unittests/state_transition.cpp

```diff
@@ -61,28 +60,37 @@ void state_transition::TearDown()
     const auto res = state::transition(state, block, tx, rev, selected_vm, block.gas_limit,
         state::BlockInfo::MAX_BLOB_GAS_PER_BLOCK);
 
-    if (const auto expected_error = make_error_code(expect.tx_error))
+    const auto expected_error = make_error_code(expect.tx_error);
```

Comment:
Why? It's never used outside of this `if` scope.

Reply:
some interim version leftovers